### ISSUE TITLE
[ACCESSIBILITY] Find a poll functionality not screen reader accessible 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1446]
**Describe the accessibility issue**
On the find a poll modal there are few issues. 
1. The text "Enter the four digit poll number to see the poll." is not announced by assistive technology. --> Please make this labeled so that assistive technology will announce it. 
2. There is an i tooltip that is not navigable by users of assistive technology. 
3. There are 4 fields, the screen reader users need these to be distinguishable from one another. 
**Please rename fields to be:** 
"Edit box 1of 4" "Edit box 2 of 4" "Edit box 3 of 4" "Edit box 4 of 4"   

[WCAG 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)

**To Reproduce
Steps to reproduce the behavior:**
**(With Screen reader on, TalkBack or VoiceOver)**
1. Launch app
2. Click 'Browse'
3. Click 'Quick Polls'
4. Click 'Find poll'

Expected behavior
Modal pops up and screen reader users are alerted what it is for ""Enter the four-digit poll number to see the poll. i.  Edit box 1 of 4"
Actual behavior
Modal pops up and screen reader users hear "edit box" "edit box" "edit box" not knowing how many to complete or even why because they are given no context. 

Screenshots

![Screenshot_20220310-130437](https://user-images.githubusercontent.com/82892873/157738650-27cecb9b-0ef5-46cd-bab4-e2c62cf1ffd0.png)

Smartphone (please complete the following information):

iPhone XR- iOS 14.6, Pixel 4a- Android 12

Additional context

**Comment by acovher2** [comment]
in version 3.2.37 this is resolved. 
Thank you! :)
